tests/network/hotplug.go

```diff
@@ -182,35 +154,6 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
-	})
-
-	Context("a running VMI with reserved interfaces resources", func() {
```

Comment:
Do we have another test hotplugging more than 1 interface?

Reply:
Yes, its is not removed (line 215). The remove part is the setup fixture.